services/app-web/src/pages/App/AppRoutes.tsx

```diff
@@ -80,24 +80,24 @@ const StateUserRoutes = ({
                 />
                 <Route element={<SubmissionSideNav />}>
                     {showQuestionResponse && (
-                        <Route
-                            path={
-                                RoutesRecord.SUBMISSIONS_QUESTIONS_AND_ANSWERS
-                            }
-                            element={<QuestionResponse />}
-                        />
+                        <>
+                            <Route
+                                path={
+                                    RoutesRecord.SUBMISSIONS_QUESTIONS_AND_ANSWERS
+                                }
+                                element={<QuestionResponse />}
+                            />
+                            <Route
+                                path={RoutesRecord.SUBMISSIONS_UPLOAD_RESPONSE}
+                                element={<UploadResponse />}
+                            />
+                        </>
                     )}
                     <Route
                         path={RoutesRecord.SUBMISSIONS_SUMMARY}
                         element={<SubmissionSummary />}
```

Comment:
@JasonLin0991 Another route I think could eventually be in "sidebar (aka the locked package submission root) -is the previous revision page. I noticed right now it does its own fetch but we could move to outlet and also have it share the fetch for submission summary which includes revisions. thoughts? I would do that kind of work in a separate add-on PR

Reply:
Sounds good!